static/js/event/event.ts

```diff
@@ -18,14 +18,49 @@
  * Disaster events
  */
 
+import axios from "axios";
 import React from "react";
 import ReactDOM from "react-dom";
 
-import { Page } from "./page";
+import { EventPage, Page } from "./page";
+import { Property } from "./types";
+
+function parseApiResponse(resp): Array<Property> {
```

Comment:
when defining functions, we want to include types for the arguments. So we want `function parseApiResponse(resp: <Type for resp>) ...`

nit: parseTriplesApiResponse because you might end up using more apis and we want to know which api response this function is parsing.